### ISSUE TITLE
Redlinking

### ISSUE THREAD

**Issue by stuartpb** [stuartpb/w5-graph#1]
Part of the thing that's making this so impossible for me to write - really, the problems that projects to encapsulate the whole of anything _always_ run into - is that I find myself saying "ooh, I should add that", I write twelve new links on that thing's entry, and then five of them (charitably - it's usually more like 11) don't have existing links.

Because this is, essentially, a wiki, it can't scale out if it has a requirement that every link has to be defined before you can cut a build. That doesn't work with the way I want to write. What'd be better would be if there was a special signifier on nodes that are only found _from other nodes links' to them_, without a definition for the node itself, that tells the reader "The project doesn't actually have a message for this yet! Please contribute one!"

In other words, rels need to include **red links** (in the Wikipedia sense).

**Comment by stuartpb** [comment]
This can result in the exposure of issues where I've accidentally not matched the correct name for a node. **Good.** That's part of the value in having them exposed as dangling definitions. (I _could_ do redirects in the data, but I think I'd rather not, at least not yet. Although, kind of.)
